### PR TITLE
Port to Gtk+3

### PR DESCRIPTION
**Pull request by yashagrawal3**
@quozl  please review!

### CODE REVIEW

**Comment by chimosky** [comment]
@yashagrawal3, it'll be great if you check for flake8.

**Comment by yashagrawal3** [comment]
@chimosky  since those flake8 errors were not introduced in my pull request, so that can be taken care of separately. I will work on it later.

**Comment by quozl** [comment]
Please merge with upstream repository https://github.com/lesivan/GeoTonky as it has changes which were made after the files in this repository.  See https://github.com/sugarlabs/geotonky-activity/issues/3

Please add to your checklist to search GitHub and elsewhere for the upstream repository, in case, as it is the case again here, the repository you first look at is out of date.

Yes, flake8 can be separate, it helps with code review if it is separate.

File: sugargame/canvas.py [hunk 1]
```diff
@@ -1,71 +1,91 @@
 import os
-import gtk
-import gobject
+import gi
+gi.require_version('Gtk', '3.0')
```
Comment (quozl) [inline sugargame/canvas.py]: `gi.require_version` should be called in main program, not in sugargame.

File: .gitignore [hunk 2]
```diff
@@ -0,0 +1,2 @@
+*.pyc
+*.pyo
```
Comment (quozl) [inline .gitignore]: New line mark missing from end of file, test with `cat`.

**Comment by yashagrawal3** [comment]
Changes were merged in **https://github.com/sugarlabs/GeoTonky/pull/1**, closing this pull request.

**Pull request closed without merging**